pipeline/pipelines/code_generation_pipeline.py

```diff
@@ -103,10 +116,16 @@ def do_build_flow(self, **kwargs):
                  format_tasks.PythonFormatTask('PythonFormat', inject=kwargs),
                  gapic_tasks.GapicMergeTask('GapicMerge', inject=kwargs),
                  gapic_tasks.GapicPackmanTask('GapicPackman', inject=kwargs))
+        if kwargs['publish_env'] != "none":
```

Comment:
I would also prefer that publishing was based on the existence/absence of the `publish_env` flag, rather than needing to set the flag to "none" to disable.